frontend/src/scenes/project/Settings/index.tsx

```diff
@@ -21,8 +21,46 @@ import { commandPaletteLogic } from 'lib/components/CommandPalette/commandPalett
 import { userLogic } from 'scenes/userLogic'
 import { JSBookmarklet } from 'lib/components/JSBookmarklet'
 
-export const Setup = hot(_Setup)
-function _Setup(): JSX.Element {
+function DisplayName(): JSX.Element {
+    const { currentTeam, currentTeamLoading } = useValues(teamLogic)
+    const { patchCurrentTeam } = useActions(teamLogic)
+
+    const [name, setName] = useState(currentTeam?.name || '')
+
+    if (currentTeam?.is_demo) {
+        return (
+            <p>
+                <i>The demo project cannot be renamed.</i>
+            </p>
+        )
+    }
+
+    return (
+        <div>
+            <Input
+                value={name}
+                onChange={(event) => {
+                    setName(event.target.value)
+                }}
+                style={{ maxWidth: '40rem', marginBottom: '1rem', display: 'block' }}
+            />
+            <Button
+                type="primary"
+                onClick={(e) => {
```

Comment:
UX nit: support submitting the form with enter key.

Reply:
Actually this is one place where I'd feel more comfortable having to press Tab and then Enter to confirm. This shouldn't be as smooth as submitting a form, to avoid accidents